examples/external-oauth/README.md

```diff
@@ -16,6 +23,8 @@ This example uses `jupyterhub.services.HubOAuthenticated` to authenticate reques
 2. launch the whoami service:
```

Comment:
For `whoami-oauth`, use:
 
    bash launch-service.sh &

Or for `whoami-oauth-basic`, use:

    bash launch-service-basic.sh &

Reply:
Line 48 below: I would to minimize confusion add (`whoami-oauth`) since the process I believe would be different if using without inheritance. Perhaps:

...JupyterHub as an OAuth provider, `whoami-oauth`, are...